requirements.txt

```diff
@@ -1,2 +1,3 @@
 django==2.2.12
-uuid
\ No newline at end of file
+uuid
+enum34
```

Comment:
You don't need this. It's for older _python_ versions. Pretty sure it can mess up newer versions (3.4+)

Reply:
It just couldn't import enum so....